test/remote/gateways/remote_vanco_test.rb

```diff
@@ -34,6 +36,37 @@ def test_successful_purchase_with_ip_address
     assert_equal 'Success', response.message
   end
 
+  def test_successful_purchase_with_existing_session_id
+    previous_login_response = @gateway.send(:login)
+
+    response = @gateway.purchase(
+      @amount,
+      @credit_card,
+      @options.merge(
+        session_id: { id: previous_login_response.params['response_sessionid'], created_at: Time.now }
```

Comment:
Where does the created_at value come from? Is it equivalent to the `auth_requesttime` returned in the login request?

Reply:
I was expecting `created_at` be passed into the request by the client.

Btw, I'm also open to checking the validity of the session id on our side, if you have thoughts on that.

Reply:
Right, I guess I was thinking we would return all the required data for the `session_id` hash in the previous transaction response, to simplify the client passing them into an existing_session_id request, but I wanted to see if this assumption matched your thinking.